x/ethbridge/keeper/keeper.go

```diff
@@ -143,25 +144,31 @@ func (k Keeper) ProcessBurn(ctx sdk.Context, cosmosSender sdk.AccAddress, msg *t
 
 // ProcessLock processes the lockup of cosmos coins from the given sender
 func (k Keeper) ProcessLock(ctx sdk.Context, cosmosSender sdk.AccAddress, msg *types.MsgLock) error {
-	coins := sdk.NewCoins(sdk.NewCoin(msg.Symbol, msg.Amount), sdk.NewCoin(types.CethSymbol, msg.CethAmount))
 	logger := k.Logger(ctx)
-
-	err := k.bankKeeper.SendCoinsFromAccountToModule(ctx, cosmosSender, types.ModuleName, coins)
-
-	if err != nil {
-		logger.Error("failed to transfer coin from account to module.",
-			errorMessageKey, err.Error())
-		return err
-	}
+	var coins sdk.Coins
 
 	if k.IsCethReceiverAccountSet(ctx) {
 		coins = sdk.NewCoins(sdk.NewCoin(types.CethSymbol, msg.CethAmount))
-		err = k.bankKeeper.SendCoinsFromModuleToAccount(ctx, types.ModuleName, k.GetCethReceiverAccount(ctx), coins)
+
+		err := k.bankKeeper.SendCoins(ctx, cosmosSender, k.GetCethReceiverAccount(ctx), coins)
 		if err != nil {
-			logger.Error("failed to transfer ceth from module to account.",
+			logger.Error("failed to send ceth from account to account.",
 				errorMessageKey, err.Error())
```

Comment:
perhaps here as well, provide more info about each 'account'

Reply:
This should be visible in the tx hash. The error should also give us information we need.